c7n/resources/appelb.py

```diff
@@ -508,16 +508,15 @@ class AppELBListenerFilterBase:
     """ Mixin base class for filters that query LB listeners.
     """
     permissions = ("elasticloadbalancing:DescribeListeners",)
+    retry = staticmethod(QueryResourceManager.retry)
```

Comment:
don't need to redefine this, just use self.manager.retry

Reply:
Addressed!